src/dmd/expressionsem.d

```diff
@@ -6287,6 +6287,42 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         result = e.expressionSemantic(sc);
     }
 
+    /*
+     * Get the expression initializer for a specific struct
+     *
+     * Params:
+     *  sd = the struct for which the expression initializer is needed
+     *  loc = the location of the initializer
+     *  sc = the scope where the expression is located
+     *  t1 = the type of the expression
+     *
+     * Returns:
+     *  The expression initializer
+     */
+    private Expression getInitExp(StructDeclaration sd, Loc loc, Scope* sc, Type t)
```

Comment:
I wonder if functions like this should actually be member functions of the class.

Reply:
It could be made a nested function inside visit(AssignExp)